easybuild/framework/easyconfig/types.py

```diff
@@ -434,6 +434,11 @@ def to_checksums(checksums):
                 res.append(tuple(checksum))
             else:
                 res.append(to_checksums(checksum))
+        elif isinstance(checksum, dict):
+            d = {}
+            for k, v in checksum.items():
+                d[k] = to_checksums(v)
+            res.append(d)
```

Comment:
Please avoid using single-letter variables outside of list comprehensions. `k` should be `key`, `v` should be `value`, `d` should be `...` (not creative enough to fill in the latter myself, `checksums` is clearly not an option here)